.gitignore

```diff
@@ -2,4 +2,5 @@ node_modules
 .DS_Store
 dist
 dist-ssr
-*.local
\ No newline at end of file
+*.local
+.env
```

Comment:
@andrew-waller  @andrewhwaller `.env` wasn't in the `.gitignore`!

Reply:
Ah, good catch. I hadn't put a `.env` file in the repo by default, so it didn't get added.